config/management/genesis/src/layout.rs

```diff
@@ -66,7 +66,7 @@ impl SetLayout {
             .config
             .load()?
             .override_shared_backend(&self.backend.shared_backend)?;
-        let mut storage = config.shared_backend();
+        let mut storage = config.shared_backend_with_namespace(constants::COMMON_NS.to_string());
```

Comment:
Will this cause an error if `namespace` is still specified for the backend (but ignored)?

Reply:
No, in fact, if folks use the config, the namespace will always be specified. It is because of the config, that I made this in this format instead of the "hoping" a user realizes this goes in common. This makes writing symmetric with reading (genesis).

The more I read your comment, it feels like you ask two disjoint questions:
* Will it cause an error: No
* Will it be ignored: Yes